### PR TITLE
Reviewing Lines Part 7

### PR DESCRIPTION
**Pull request by ttgmichael**
Revisions on QUESTLV100 dialogue lines 6400-7000.

Let me know your thoughts! :)

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -11766,7 +11766,7 @@ ETC_20150714_011767	Yes!
 ETC_20150714_011768	Control the Magic Generating Stones to stop them all!
 ETC_20150714_011769	Magic Generating Stones
 ETC_20150714_011770	Archon has been released and abducted Jane!{nl}Defeat Archon and save Jane!
-ETC_20150714_011771	The Magic Stone of Interaction
+ETC_20150714_011771	The Magic Stone of Sympathy
```
Comment (Sunaries) [inline ETC.tsv]: Magic Stone of X

Interaction is possible. What is the context?

File: ETC.tsv [hunk 2]
```diff
@@ -12091,68 +12091,68 @@ ETC_20150714_012092	{memo X}Property:1/PropertyAdd:1/Func:SCR_REMAINS37_3_SQ_040
...
-ETC_20150714_012100	Check the outer wall entrance
-ETC_20150714_012101	Eliminate the weak Stone Frosts at the great square
-ETC_20150714_012102	Rub the fragrant stone powders onto Rambear
-ETC_20150714_012103	Use Monster Stimulant
-ETC_20150714_012104	Lure the guard at Watermill Ruins
-ETC_20150714_012105	Put it at Estuary three ways
-ETC_20150714_012106	Put it at Zmone Square
-ETC_20150714_012107	Put it at Badova Dock
-ETC_20150714_012108	Put it at Viska Sanctum Ruins
+ETC_20150714_012096	$Test the Symbol of Silt
```
Comment (Sunaries) [inline ETC.tsv:12094]: 실트 can also be Schilt. You would have to ask @imcgames behind the meaning of this before making this change.

File: ETC.tsv [hunk 3]
```diff
@@ -12091,68 +12091,68 @@ ETC_20150714_012092	{memo X}Property:1/PropertyAdd:1/Func:SCR_REMAINS37_3_SQ_040
...
-ETC_20150714_012125	The reason you've come here
-ETC_20150714_012126	The way to be released
-ETC_20150714_012127	Stop the 1st Magic Generating Stone
-ETC_20150714_012128	Stop the 2nd Magic Generating Stone
-ETC_20150714_012129	Stop the 3rd Magic Generating Stone
-ETC_20150714_012130	Stop all the Magic Generating Stones
-ETC_20150714_012131	Rubbing of the petrified book
+ETC_20150714_012113	$Kareras' Memoir
+ETC_20150714_012114	$Finishing the research
+ETC_20150714_012115	$Red Spatial Magic Stone(1)
```
Comment (Sunaries) [inline ETC.tsv]: space between 'Stone' and '(1)'

File: ETC.tsv [hunk 4]
```diff
@@ -12091,68 +12091,68 @@ ETC_20150714_012092	{memo X}Property:1/PropertyAdd:1/Func:SCR_REMAINS37_3_SQ_040
...
+ETC_20150714_012116	$Red Spatial Magic Stone(2)
```
Comment (Sunaries) [inline ETC.tsv]: space between 'Stone' and '(2)'

File: ETC.tsv [hunk 5]
```diff
@@ -12091,68 +12091,68 @@ ETC_20150714_012092	{memo X}Property:1/PropertyAdd:1/Func:SCR_REMAINS37_3_SQ_040
...
+ETC_20150714_012117	$Condensed Spite
+ETC_20150714_012118	$Blue Spatial Magic Stone(2)
```
Comment (Sunaries) [inline ETC.tsv]: space between 'Stone' and '(2)'

File: ETC.tsv [hunk 6]
```diff
@@ -12091,68 +12091,68 @@ ETC_20150714_012092	{memo X}Property:1/PropertyAdd:1/Func:SCR_REMAINS37_3_SQ_040
...
+ETC_20150714_012119	$Trace the location of Temple Shooter Witch at the Shrine of Remembrance
+ETC_20150714_012120	$Yellow Spatial Magic Stone(2)
```
Comment (Sunaries) [inline ETC.tsv]: space between 'Stone' and '(2)'

File: ETC.tsv [hunk 7]
```diff
@@ -12407,7 +12407,7 @@ ETC_20150714_012408	You absorbed the weak Stone Frost
 ETC_20150714_012409	$Rubbing the Scented Stone powders...
 ETC_20150714_012410	Rubbed the Scented Stone powders!
 ETC_20150714_012411	$Already rubbed in the Scented Stone powders!
-ETC_20150714_012412	Monsters have become more agile due to the effects from the symbol of Schilt
+ETC_20150714_012412	Monsters have become more agile due to the effects from the Symbol of Silt
```
Comment (Sunaries) [inline ETC.tsv:12410]: See previous note about this.

File: QUEST_LV_0100.tsv [hunk 8]
```diff
@@ -6403,51 +6403,51 @@ QUEST_LV_0100_20150317_006402	{memo X}Mr. Juan asked you to bring him Yukopus Le
...
 QUEST_LV_0100_20150323_006433	{memo X}$Death may be the price that I pay for the sin I committed. {nl}Even if it is the curse of the demons, it is still for a sin I made.
 QUEST_LV_0100_20150323_006434	{memo X}$I don't want to... die like this... {nl}Mother...
 QUEST_LV_0100_20150323_006435	$Wait. Aren't those Succubus Eyes? {nl}Wow. How did you get them... Can I take a look? 
-QUEST_LV_0100_20150323_006436	$I've heard about those eyes. {nl}The Succubus craved the Great Cathedral's blessing that was hidden in a necklace, so she put it in her eyes. {nl}
-QUEST_LV_0100_20150323_006437	$Did you know I used to be an aspiring Alchemist? {nl}I can turn that back into a necklace. {nl}
-QUEST_LV_0100_20150323_006438	$Done. {nl}Do you know about the five necklaces of Maven?{nl}
+QUEST_LV_0100_20150323_006436	$I've heard about those eyes. {nl}A greedy Succubus tried to take one of the Great Cathedral's blessed necklaces, but ended up{nl}putting her own eyes into this one. {nl} 
+QUEST_LV_0100_20150323_006437	$Did you know I used to be an aspiring Alchemist? {nl}I can turn that back into a normal necklace. {nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:6437]: Alchemist -> alchemist

File: QUEST_LV_0100.tsv [hunk 9]
```diff
@@ -6914,90 +6914,90 @@ QUEST_LV_0100_20150511_006913	It seems that the newcomer seems to be in a troubl
...
+QUEST_LV_0100_20150714_006988	$The magic generating stones in the Room of Ancestral Rituals affect the other magic generating {nl}stones nearby. We could start over with Valius' magic.
+QUEST_LV_0100_20150714_006989	$The magical power supplied to the Magic Stone of Sympathy has been completely {nl}shut off... Is it alright to wish for the salvation now?
+QUEST_LV_0100_20150714_006990	$We are all ready. But that Archon is different from the Archon from your era. {nl}It may be better to leave it as is.
+QUEST_LV_0100_20150714_006991	$There's is another way. The sealed keepsakes amongst Valius's collection of keepsakes...{nl}I heard some of them were created in Uzdaromo Stone Chamber to suppress the demons.
+QUEST_LV_0100_20150714_006992	$I had already given up the hope that I would become free a few hundreds years ago.{nl}Even now, I have little hope for it.
+QUEST_LV_0100_20150714_006993	$The seal that is suppressing the demonic energy... that's it.{nl}Now everything is ready.
+QUEST_LV_0100_20150714_006994	$Now when the Magic Stone of Sympathy stops, I will be free from Archon.{nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:6994]: See previous note about this.

File: QUEST_LV_0100.tsv [hunk 10]
```diff
@@ -7422,35 +7422,35 @@ QUEST_LV_0100_20150714_007421	Get Matsum's Flower Stamen to create a Thorn Flowe
...
-QUEST_LV_0100_20150714_007433	You've found a way to block the magical energy of the magical stone of the interaction from the records of Valius. Return to Jane's spirit and let her know.
-QUEST_LV_0100_20150714_007434	According to the information on the magical stone of the interactionn which Valius left, there's a way to save Jane. Talk to Jane about it.
-QUEST_LV_0100_20150714_007435	Search the keepsakes which possess the magical power of Valius
-QUEST_LV_0100_20150714_007436	Jane's spirit told you that in order to stop magic generating stones that is supplying the magical power to the magical stone of the interaction, Valius' magical power is needed. Look for the objects that are likely to possess Valius magical power at Vebaiki Burial Chamber.
+QUEST_LV_0100_20150714_007425	$Talk to Jane's Spirit
+QUEST_LV_0100_20150714_007426	$Jane's Spirit is not parting from the dangerous Archon. Ask her why.
+QUEST_LV_0100_20150714_007427	$Listen to Jane's Spirit
+QUEST_LV_0100_20150714_007428	$It seems that something complicated is going on with Jane. Listen from Jane's Spirit what's wrong.
+QUEST_LV_0100_20150714_007429	$The Void that Arisen
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:7429]: that arisen? or arose?

File: ETC.tsv (hunk 1 above)
Comment (ttgmichael) [inline ETC.tsv]: It seems to be a quest about releasing Jane's Spirit from an Archon. The Magic Stone seems to be binding the spirit to the monster thus the spirit cannot be freed unless you stop the magic coming out of the Magic Stone (and defeat Archon later).

I guess Interaction works, but it seems to be a bland word for "magic that binds the two together". So I was thinking either Sympathy or Mutuality (to make it seem like both are being affected by a certain condition).

File: QUEST_LV_0100.tsv (hunk 10 above)
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:7429]: my bad, will change xD

File: ETC.tsv (hunk 2 above)
Comment (ttgmichael) [inline ETC.tsv:12094]: @imcgames what's your opinion on 실트 in (실트의 증표)? I simply kept it consistent with the other 실트의 증표 in the other files, which is named Symbol of Silt. ^^

File: ETC.tsv (hunk 1 above)
Comment (Sunaries) [inline ETC.tsv]: Mutuality would work.

File: ETC.tsv (hunk 2 above)
Comment (Sunaries) [inline ETC.tsv:12094]: I've asked IMC. Their reponse was that it didn't had any significant meaning. So, it's your pass.

Comment (ttgmichael) [inline ETC.tsv:12094]: I'll keep it consistent with the other files for now :+1:

File: ETC.tsv (hunk 1 above)
Comment (ttgmichael) [inline ETC.tsv]: :+1:

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
Thank :+1:

**Pull request merged**